examples/HelloWorldTriangle/triangle.ts

```diff
@@ -3,28 +3,28 @@
  */
 
 import {
-  WebGLShader, shaderSource, createShader, compileShader, 
+  WebGLShader, shaderSource, createShader, compileShader,
   VERTEX_SHADER, FRAGMENT_SHADER, createProgram, WebGLProgram,
   attachShader, useProgram, WebGLUniformLocation, getUniformLocation,
   linkProgram, clearColor, clear, WebGLBuffer,
   createBuffer, ARRAY_BUFFER, GLint,
   STATIC_DRAW, FLOAT, COLOR_BUFFER_BIT,
   enableVertexAttribArray, bindBuffer, createContextFromCanvas,
-  bufferData, getAttribLocation, drawArrays, 
+  bufferData, getAttribLocation, drawArrays,
   vertexAttribPointer, TRIANGLE_STRIP,
 } from '../../webgl'
 
-const VERTEX_SHADER_CODE:string = `#version 300 es
+const VERTEX_SHADER_CODE:string = /*glsl*/ `#version 300 es
```

Comment:
My VS Code extensions will highlight the glsl syntax within the template strings based on the syntax comment.

![Uploading syntax.png…]()